vm/vm/src/createTevm.js

```diff
@@ -58,10 +56,10 @@ export const createTevm = async (options = {}) => {
 		const blockTag = options.fork.blockTag ?? (await client.getBlockNumber())
 		stateManager = new TevmStateManager({ client, blockTag })
 	} else {
-		stateManager = new DefaultStateManager()
+		stateManager = new DefaultTevmStateManager()
 	}
 
-	const evm = new _EVM({
+	const evm = new TevmEvm({
```

Comment:
To add the handler to Tevm you simply need to add the type to the Tevm type here: https://github.com/evmts/tevm-monorepo/blob/main/vm/api/src/Tevm.ts and then in this file add your handler to it

const tevm = {
 ...
  dumpState: dumpStateHandler(stateManager)
...
}